src/Symfony/Component/Routing/Tests/Fixtures/AnnotationFixtures/FooController.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Symfony\Component\Routing\Tests\Fixtures\AnnotationFixtures;
+
+use Symfony\Component\Routing\Annotation\Route;
+
+class FooController
+{
+    /**
+     * @Route("/Blog")
+     */
+    public function simplePath()
+    {
+    }
+
+    /**
+     * @Route({"nl":"/hier","en":"/here"})
+     */
+    public function localized()
+    {
+    }
+
+    /**
+     * @Route("", requirements={"locale":"en"})
```

Comment:
```suggestion
     * @Route(requirements={"locale":"en"})
```

There's no need to pass a path.